### PR TITLE
PEP8 fixes in datasets/*

### PR DESCRIPTION
**Pull request by banilo**
Cleanups from yesterday...

### CODE REVIEW

File: nilearn/datasets/utils.py [hunk 1]
```diff
@@ -719,8 +719,8 @@ def _fetch_files(data_dir, files, resume=True, mock=False, verbose=1):
         files_.append(target_file)
     # If needed, move files from temps directory to final directory.
     if os.path.exists(temp_dir):
-        #XXX We could only moved the files requested
-        #XXX Movetree can go wrong
+        # XXX We could only moved the files requested
```
Comment (dohmatob) [inline nilearn/datasets/utils.py]: moved the files requested ==> move the requested files

File: nilearn/datasets/utils.py [hunk 2]
```diff
@@ -719,8 +719,8 @@ def _fetch_files(data_dir, files, resume=True, mock=False, verbose=1):
...
+        # XXX Movetree can go wrong
```
Comment (dohmatob) [inline nilearn/datasets/utils.py]: ? This XXX looks dangerous. It'd be better to tackle the underlying problem.

File: nilearn/datasets/tests/test_struct.py [hunk 3]
```diff
@@ -118,8 +118,8 @@ def test_get_dataset_dir():
     expected_base_dir = os.path.join(tmpdir, 'nilearn_shared_data')
     os.environ['NILEARN_SHARED_DATA'] = expected_base_dir
     data_dir = utils._get_dataset_dir('test',
-                                               default_paths=[no_write],
-                                               verbose=0)
+                                      default_paths=[no_write],
+                                      verbose=0)
     # Non writeable dir is returned because dataset may be in there.
     assert_equal(data_dir, no_write)
     assert os.path.exists(data_dir)
```
Comment (dohmatob) [inline nilearn/datasets/tests/test_struct.py]: Use assert_true instead.

**Comment by AlexandreAbraham** [comment]
LGTM

**Comment by lesteve** [comment]
Merging, thanks!

**Comment by banilo** [comment]
:octocat: Good idea!

2015-07-19 20:09 GMT+02:00 Loïc Estève <notifications@github.com>:

> Merged #710 <https://github.com/nilearn/nilearn/pull/710>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nilearn/nilearn/pull/710#event-359503943>.
>



-- 
Viele Grüße,
Danilo

**Pull request merged**